dist/offline_installer/redhat/build_offline_installer.sh

```diff
@@ -92,22 +56,19 @@ if [ ! -f /usr/bin/createrepo ]; then
     sudo yum -y install createrepo
 fi
 
-sudo yum -y install yum-plugin-downloadonly
-
-cd /etc/yum.repos.d/
-sudo wget -N $REPO
-cd -
-
 sudo rm -rf build/installroot build/offline_installer build/scylla_offline_installer.sh
 mkdir -p build/installroot
 mkdir -p build/installroot/etc/yum/vars
 sudo sh -c "echo $RELEASE >> build/installroot/etc/yum/vars/releasever"
 
 mkdir -p build/offline_installer
 cp dist/offline_installer/redhat/header build/offline_installer
-if [ -n "$RELEASEVER" ]; then
-    YUMOPTS="--releasever=$RELEASEVER"
-fi
-sudo yum -y install $YUMOPTS --downloadonly --installroot=`pwd`/build/installroot --downloaddir=build/offline_installer scylla sudo ntp ntpdate net-tools kernel-tools
+cat <<EOF > build/construct_offline_repo.sh
+yum -y install epel-release wget yum-plugin-downloadonly
+wget $REPO -O /etc/yum.repos.d/scylla.repo
+yum -y install --releasever=$RELEASE --downloadonly --installroot=`pwd`/build/installroot --downloaddir=build/offline_installer scylla sudo ntp ntpdate net-tools kernel-tools
+EOF
+
+sudo ./tools/toolchain/dbuild --image centos:7 -- bash ./build/construct_offline_repo.sh
```

Comment:
why sudo?

Reply:
Use docker.io/centos:7 so it works better with podman (podman looks for other registries first)

Reply:
root user was needed to run yum install, but I find a way to avoid taking root privilege here:
 - don't install package on container running time, build custom docker image instead
 - add fakeroot on `yum install --downloadonly`